Sources/Socket.swift

```diff
@@ -193,19 +197,21 @@ public struct SocketEvent : OptionSet {
 extension Socket {
     func setOption(_ option: Int32, _ value: Bool) throws {
         var value = value ? 1 : 0
-        try setOption(option, value: &value, length: strideof(Int32))
+        try setOption(option, value: &value, length: MemoryLayout<Int32>.stride)
```

Comment:
I believe you can use `size` instead of `stride` here, since there's only one element.